### ISSUE TITLE
Chat has strange UI with screen reader 

### ISSUE THREAD

**Issue by 88572** [element-hq/element-x-android#3764]
### Steps to reproduce

1. Go to a chat
2. Have someone send 2 or more messages 
3. Turn on screen reader
4. Observe that the UI looks strange 

#### What did you expect?
Messages should have names on top of chains and rounded corners should be as normal
#### What happened instead?
![Screenshot_20241030-163706](https://github.com/user-attachments/assets/394b65eb-c334-4308-94e3-29de2553894c)

### Your phone model

Google Pixel 8a

### Operating system version

Android 15

### Application version and app store

Element X 0.7.2 from Google Play

### Homeserver

sulian.eu, Synapse 1.118.0

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
This is a workaround for a bug found in the accessibility implementation of Compose (the UI framework), which completely broke scrolling when the 'reversed' layout mode is enabled, where everything is placed from the bottom to the top. We tried several workarounds but the only thing that seemed to work was removing the 'reversed' flag, which breaks the UI but makes the timeline usable for people who can't rely on their sight.

We could try removing this now, since google may have fixed the bug that caused the broken scrolling behaviour.

**Comment by jmartinesp** [comment]
> We could try removing this now, since google may have fixed the bug that caused the broken scrolling behaviour.

Sadly, this is still broken...
